Compiling FreeRDP for MACOSX

Hey,

Where can I find the latest documentation to compile FreeRDP Mac OSX.

I noticed the instructions are for mountain lion?

thanks

@Mandar-Shinde 
ran, cmake -D "CMAKE_OSX_ARCHITECTURES:STRING=i386;x86_64" . && make

getting these errors?

Linking C shared library libwinpr2.dylib
Undefined symbols for architecture i386:
  "_ERR_remove_thread_state", referenced from:
      _winpr_CleanupSSL in ssl.c.o
  "_FIPS_mode", referenced from:
      __winpr_openssl_initialize in ssl.c.o
      _winpr_FIPSMode in ssl.c.o
  "_FIPS_mode_set", referenced from:
      __winpr_openssl_initialize in ssl.c.o
ld: symbol(s) not found for architecture i386
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [winpr/libwinpr/libwinpr2.2.0.0.dylib] Error 1
make[1]: *** [winpr/libwinpr/CMakeFiles/winpr.dir/all] Error 2
make: *** [all] Error 2

--- a/libfreerdp/crypto/crypto.c
+++ b/libfreerdp/crypto/crypto.c
@@ -756,7 +756,6 @@ char* crypto_cert_issuer(X509* xcert)
 
 BOOL x509_verify_certificate(CryptoCert cert, char* certificate_store_path)
 {
-	int purpose;
 	X509_VERIFY_PARAM* verify_param;
 	X509_STORE_CTX* csc;
 	BOOL status = FALSE;
@@ -802,13 +801,7 @@ BOOL x509_verify_certificate(CryptoCert cert, char* certificate_store_path)
 	if (!X509_STORE_CTX_init(csc, cert_ctx, xcert, cert->px509chain))
 		goto end;
 
-	purpose = X509_PURPOSE_ANY;
-	verify_param = X509_STORE_CTX_get0_param(csc);
-
-	if (!verify_param)
-		goto end;
-
-	X509_VERIFY_PARAM_set_purpose(verify_param, purpose);
+	X509_STORE_CTX_set_purpose(csc, X509_PURPOSE_SSL_SERVER);
 
 	if (X509_verify_cert(csc) == 1)
 		status = TRUE;

